src/PoolLike/PoolLike.types.ts

```diff
@@ -0,0 +1,23 @@
+export type PoolLikeAttributes = {
+  pool_id: string
+  user_id: string
+  created_at: string
+}
+
+export type PoolLikeCount = {
+  pool_id: string
+  user_id?: string
+}
+
+export const poolLikeSchema = {
```

Comment:
Should we delete `poolLikeSchema`? it's not really being used. Or should we use it in the corresponding endpoint?